docs/src/pages/components/stepperfield/index.page.mdx

```diff
@@ -1,5 +1,6 @@
 ---
 title: StepperField
+description: A StepperField allows to increase or decrease a numeric value
```

Comment:
What do you think about, 'A StepperField is a number input with buttons to increase or decrease the value.' ? (I know the original text was taken from a document I wrote, so that is my bad)